Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.DotNet.Interactive.ExtensionLab.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+ <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <PackageDescription>Experimental kernel extensions for dotnet-interactive</PackageDescription>
+    <PackageTags>dotnet-interactive</PackageTags>
+  </PropertyGroup>
+
```

Comment:
here you need something like
```
  <ItemGroup>
    <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="interactive-extensions/dotnet" Visible="false" />
  </ItemGroup>
```

The path ```interactive-extensions/dotnet``` is the location the composite kernel uses for probing